src/shims/windows/handle.rs

```diff
@@ -35,6 +35,14 @@ impl PseudoHandle {
     }
 }
 
+/// Handle errors
+pub enum HandleError {
+    /// There is no thread with the given ID
+    ThreadNotFound(ThreadNotFound),
+    /// Can't convert scalar to handle because it is structurally invalid
```

Comment:
```suggestion
    /// There is no thread with the given ID.
    ThreadNotFound(ThreadNotFound),
    /// Can't convert scalar to handle because it is structurally invalid.
```